docs/source/api/touch.rst

```diff
@@ -0,0 +1,126 @@
+#####
+TOUCH
+#####
+
+About
+-----
+
+Touch sensor is a peripheral, that can be used to sense electrical changes on respective GPIO pins. Therefore these touch sensors 
+are also known as capacitive sensors. For example, if you touch any of these pins, the touch sensor will sense output according to 
+electrical charge on your finger. These pins can be easily integrated into capacitive pads, and replace mechanical buttons.
+
+.. note:: Touch peripheral is not present in every SoC. Refer to datasheet of each chip for more info.
+
+Arduino-ESP32 TOUCH API
+-----------------------
+
+TOUCH common API
+****************
+
+touchRead
+^^^^^^^^^
+
+This function is used to get the TOUCH pad value for a given pin.
+
+.. code-block:: arduino
+
+    touch_value_t touchRead(uint8_t pin);
+
+* ``pin`` GPIO pin to read TOUCH value
+  
+This function will return touch pad value as uint16_t (ESP32) or uint32_t (ESP32-S2/S3).
+
+touchSetCycles
+^^^^^^^^^^^^^^
+
+This function is used to set cycles that measurement operation takes. The result from touchRead, threshold and detection accuracy depend on these values.
+Defaults are 0x1000 for measure and 0x1000 for sleep. With default values touchRead takes 0.5ms.
```

Comment:
Are the values the same?

Reply:
Fixed, defaults are different for each chip. Mentioned only the time it takes the touch sensor to read.